### DISCUSSION TITLE
Metadata Variables for output/record

### DISCUSSION THREAD

**Discussion by Veatra** [streamlink/streamlink#5187] [Q&A] [answered]
### Description

Just a quick questions on the metatags.

Anyway to get streamlink just to output the content of {category} etc. directly?

Is it possible to get the original upload date of a vod instead of the time of starting the vod download?

**Comment by bastimeyer** [comment, marked as answer]
> get streamlink to output the content of {category} etc. directly?

https://streamlink.github.io/cli.html#cmdoption-json
https://stedolan.github.io/jq/

```sh
streamlink --json "$STREAMURL" "$QUALITY" | jq -r .metadata.category
```

> Is it possible to get the original upload date of a vod instead of the time of starting the vod download?

There is no such metadata for that. This also wildly depends on the streaming site you're trying to access.

If you're talking about Twitch for example, then just use their public helix API or their private gql API yourself, which you should be doing for reading any kind of metadata anyway when doing additional scripting, as Streamlink isn't really meant for this.
